modules/seo/helpers-seo.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Used to check whether advanced seo features are enabled.
+ *
+ * @return bool True if advanced seo features are enabled, false otherwise.
+ */
+function is_enabled_advanced_seo() {
+	// stub
+	return true;
+}
+
+/**
```

Comment:
> I removed that in order to preserve backwards compatibility with PHP 5.2 (is it still a requirement?).

That's a good call. Jetpack should be compatible with PHP 5.2+, so that means no namespaces. But putting everything in a class works 👍 . Whether you want to do that or just prefix the functions is up to you.